test/paper-spinner.html

```diff
@@ -54,6 +54,25 @@
             spinner.active = false;
             expect(spinner.getAttribute('aria-hidden')).to.be.eql('true');
           });
+
+          test('toggle during cooldown', function(done) {
+            spinner.active = true;
+
+            window.requestAnimationFrame(function() {
```

Comment:
I want the spinner's observers to fire and actually change `active` to true before then setting it back. Maybe it would be better to call `spinner.async`? I get the feeling that `async` is intended to be 'protected' though.